vivarium/core/experiment.py

```diff
@@ -1349,7 +1351,8 @@ def emit_configuration(self):
             'data': data}
         try:
             self.emitter.emit(emit_config)
-        except:
+        except Exception:  # TODO(jerry): Narrow this too broad exception
+            log.exception("emitter.emit", exc_info=True, stack_info=True)
```

Comment:
The reason I added this `try`/`except` is that some processes have enormous configs that are over mongoDB's limit -- in particular if they contain gene sequence data. So if it can't be emitted, we remove the `'process'` and `'state'` configuration data, and emit the rest.

Reply:
Which specific exceptions can that raise? I'll change this to `PyMongoError`, which is a big improvement, e.g. it won't swallow an `AttributeError` indicating a bug.

This is probably sufficient but narrowing it further (to `WTimeoutError` and/or `DocumentTooLarge`?) would not give a pass to a bug such as a timeout trying to contact the server.

Reply:
`PyMongoError` sounds good to me!